3648/1/output.rb

```diff
@@ -0,0 +1,53 @@
+require 'terminal-table'
+require_relative 'gem_stat'
+
+HEADERS = %w[Name Used\ by Watched\ by Stars Forks Contributors Issues].freeze
+
+class Output
+  def initialize(options)
+    @options = options
+  end
+
+  def print_table
+    rows = []
+    select_top_gems.each { |gem| rows << gem.values }
+    puts Terminal::Table.new headings: HEADERS, rows: rows
+  end
+
+  private
+
+  def parse_flag
```

Comment:
You assign one instance variable to another one. It's useless assignment. If you want just to have different name, then name it right from the first attempt in `initialize`